egs2/mini_an4/asr1/transfer_learning.md

```diff
@@ -0,0 +1,41 @@
+## Use transfer learning for ASR in ESPnet2
+
+In that tutorial, we will introduce several options to use pre-trained models/parameters for Automatic Speech Recognition (ASR) in ESPnet. Available options are : 
+- use a local model you (or a collegue) have already trained,
+- use a trained model from ESPnet repository on HuggingFace.
+
+We note that this is done for ASR training, so at __stage 11__ of ESPnet2 models' recipe.
+
+### 0. Why using such (pre-)trained models ? 
+
+Several projects may involve making use of previously trained models, this is the reason why we developed ESPnet repository on HuggingFace for instance.
+Example of use cases are listed below (non-exhaustive):
+- target a low resource language, a model trained from scratch may perform badly if trained with only few hours of data,
+- study robustness to shifts (domain, language ... shifts) of a model,
+- use massively trained multilingual ASR models ...
+
+### 1. Use a local model that you have already trained. 
+
+__Step 1__ : make sure your ASR model file has the proper ESPnet format (should be ok if trained with ESPnet). It just needs to be a ".pth" (or ".pt" or other extension) type pytorch model.
+
+__Step 2__ : add the parameter ```--pretrained_model path/to/your/pretrained/model/file.pth``` to run.sh. 
+
+__Step 3__ : step 2 will initialize your new model with the parameters of the pre-trained model. Thus your new model will be trained with a strong initialization. However, if your new model have different parameter sizes for some parts of the model (e.g. last projection layer could be modified ...). This will lead to an error because of mismatches in size. To prevent this to happen, you can add the parameter ```--ignore_init_mismatch true``` in run.sh.
```

Comment:
```suggestion
__Step 3__ : step 2 will initialize your new model with the parameters of the pre-trained model. Thus your new model will be trained with a strong initialization. However, if your new model has different parameter sizes for some parts of the model (e.g. last projection layer could be modified ...). This will lead to an error because of mismatches in size. To prevent this to happen, you can add the parameter ```--ignore_init_mismatch true``` in run.sh.
```